lib/new-settings-loader.js

```diff
@@ -23,6 +26,30 @@ const SECRET_FIELDS = [
   'twilio.authToken'
 ]
 
+/*
+ * JSON.stringify isn't necessarily deterministic so this function may compute
+ * different hashes for the same object.
+ */
+const md5hash = obj =>
+  crypto
+    .createHash('MD5')
+    .update(JSON.stringify(obj))
```

Comment:
We just need the content hash, so you don't have stringify for the rest of the settings.